includes/class-wc-calypso-bridge-admin-menu.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * Class Ecommerce_Atomic_Admin_Menu.
+ *
+ * @since x.x.x
+ *
+ * The admin menu controller for Ecommerce WoA sites.
+ */
+class Ecommerce_Atomic_Admin_Menu extends \Automattic\Jetpack\Dashboard_Customizations\Atomic_Admin_Menu {
+
+	/**
+	 * Remove Stats menu.
+	 */
+	public function add_stats_menu() {
+		// Silence!
+	}
+
+	/**
+	 * Remove the Jetpack menu
+	 */
+	public function add_jetpack_menu() {
+
+		global $submenu;
+
+		parent::add_jetpack_menu();
+
+		// Remove Jetpack Search menu item. Already exposed in the Jetpack Dashboard.
+		$this->hide_submenu_page( 'jetpack', 'jetpack-search' );
+
+		// Move Akismet under Settings
+		$this->hide_submenu_page( 'jetpack', 'akismet-key-config' );
+		add_submenu_page( 'options-general.php', __( 'Anti-Spam', 'wc-calypso-bridge' ), __( 'Anti-Spam' , 'wc-calypso-bridge' ), 'manage_options', 'akismet-key-config', array( 'Akismet_Admin', 'display_page' ), 12 );
+
+		// Move Stats in here.
+		add_submenu_page( 'jetpack', esc_attr__( 'Stats', 'wc-calypso-bridge' ), __( 'Stats', 'wc-calypso-bridge' ), 'manage_options', 'https://wordpress.com/stats/day/' . $this->domain, null, 20 );
+
+		// Move Jetpack Status screen from Settings to Tools.
+		remove_submenu_page( 'options-general.php', 'https://wordpress.com/settings/jetpack/' . $this->domain );
+		add_submenu_page( 'tools.php', esc_attr__( 'Jetpack Status', 'wc-calypso-bridge' ), __( 'Jetpack Status', 'wc-calypso-bridge' ), 'manage_options', 'https://wordpress.com/settings/jetpack/' . $this->domain, null, 100 );
+
+		// Remove Earn from Tools.
+		$this->hide_submenu_page( 'tools.php', 'https://wordpress.com/earn/' . $this->domain );
```

Comment:
There is a similar method to `add_stats_menu` that adds the items of the tools named `add_tools_menu` see https://github.com/Automattic/jetpack/blob/42b85a31604f673b9ff1773f386bf69d0abf47e1/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L331-L346 Why not override that method here? That way, we gain more control over the menu structure. 

If you want feel like copying&pasting the same code it could be something along the lines:
```php
public function add_tools_menu() {
    parent::add_tools_menu();
    $this->hide_submenu_page( 'tools.php', 'https://wordpress.com/earn/' . $this->domain );
}
```

Reply:
Similar we could utilize the `add_options_menu` as well here: https://github.com/Automattic/jetpack/blob/42b85a31604f673b9ff1773f386bf69d0abf47e1/projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php#L351

Reply:
Done!